boskos/configs.yaml

```diff
@@ -13,6 +13,11 @@ configs:
           version: 1.9
           zone: us-central1-f
           enablekubernetesalpha: true
+        - machinetype: n1-standard-2
```

Comment:
Is modifying the current boskos resource type the short term change?  I was thinking we'd add a new element to the list for a resource type for use by multicluster tests.

Reply:
It is up to you. I was affraid we would run out of resources if we add another one. Maybe not.

Reply:
I'm not sure of the cost of new projects v. the cost of making all projs in current boskos pool get 2 gke clusters allocated all the time.  I think the 2 gke cluster allocation would be more expensive.

One other thing I noticed is that if we did make the 2nd cluster different (smaller), I think the only way we could tell which cluster was which type is to use gcloud commands.  I think if we add a name attribute here in the boskos/config.yaml we could just use `kubectl config *-context` to determine the more powerful cluster.  But if other tests need multiple identical clusters the the name attribute isn't required.